modern/package.json

```diff
@@ -17,6 +17,7 @@
     "maplibre-gl": "^1.15.0",
     "material-ui-dropzone": "^3.5.0",
     "moment": "^2.29.1",
+    "notistack": "^1.0.10",
```

Comment:
I don't like adding new library just for supporting multiple notifications. Can't we just stack default ones?

Reply:
I think we can, but we basically would have to do that notistack does. There's a [discussion about stacking snackbars](https://github.com/mui/material-ui/issues/1824).

Reply:
I would say if it's too complicated or requires too much code, let's only support a single one.

Reply:
Agreed, this was my first approach. But since we cannot control when we will receive events from websocket we need to have some kind of list or queue for the notifications that arrive.

Otherwise, if we receive an event from websocket and just update the component state it could lead to too much re-rendering if many events arrive in short time, causing error.

Reply:
Why would there be an error?

Reply:
Because an event can arrive and change the component state, generating a re-rendering just in the same time the component is being rendered.

Reply:
That's fine. Why would cause an error?

Reply:
Sorry, I might be mistaken:
> “Too many re-renderers” is a React error that happens after you have reached an infinite render loop, typically caused by code that in a useEffect hook or the main body of the component itself unconditionally calls state setters.

I'll have to try something like the first approach but keeping in mind the multiple events concern.